mlrun/projects/pipelines.py

```diff
@@ -891,15 +892,33 @@ def load_and_run(
     ttl: int = None,
     engine: str = None,
     local: bool = None,
+    schedule: typing.Union[str, mlrun.api.schemas.ScheduleCronTrigger] = None,
 ):
-    project = mlrun.load_project(
-        context=f"./{project_name}",
-        url=url,
-        name=project_name,
-        init_git=init_git,
-        subpath=subpath,
-        clone=clone,
-    )
+    try:
+        project = mlrun.load_project(
+            context=f"./{project_name}",
+            url=url,
+            name=project_name,
+            init_git=init_git,
+            subpath=subpath,
+            clone=clone,
+        )
+    except Exception as error:
+        if schedule:
+            # Notifying slack:
+            slack_notifier = mlrun.utils.notifications.SlackNotification()
+            url = get_ui_url(project_name, context.uid)
+            link = f"<{url}|*view workflow job details*>"
+            message = (
+                f":x: Failed to run scheduled workflow {workflow_name} in Project {project_name} !\n"
+                f"error: ```{error}```\n{link}"
+            )
```

Comment:
@quaark  is that syntax different from your suggestion on the future notificaitons?

Reply:
This is different. And also won't work as that send function is asynchronous. 
@yonishelach you need to use the `CustomNotificationPusher` which will abstract the asynchronous part for you.